crates/ruff/src/settings/types.rs

```diff
@@ -244,3 +282,35 @@ impl Deref for Version {
         &self.0
     }
 }
+
+#[derive(Debug, PartialEq, Eq)]
+pub struct RequiresVersion(pub Vec<VersionSpecifier>);
```

Comment:
Is it possible to make the inner `Vec` private (are there any reads-writes outside of this module)?

If not, can we use accessors to make it private, e.g. implement `Deref` with `Target =[VersionSpecifier]` or `IntoIter` on `&RequiresVersion`? 

Doing so has the benefit of not exposing the used type to store the `VersionSpecifier`. E.g. we could consider using a `SmallVec` in the future.

Reply:
good point, i've also taken this in https://github.com/konstin/pep440-rs/commit/08f3e6b2d91fa1d257cbb2675e0fc22645b2701c

Reply:
I replaced `RequiresVersion` by `VersionSpecifiers`.